src/data/index.ts

```diff
@@ -14,9 +16,11 @@ export function start(): void {
     measure(metadata.start)();
     measure(ping.start)();
     measure(tag.reset)();
+    measure(upgrade.reset)();
 }
 
 export function end(): void {
```

Comment:
Is the reverse ordering of calls in start and end important? If so, add a comment about why.

Reply:
Not entirely. Order can be mixed up for independent modules however for modules with internal dependency - e.g. -  upload takes dependency on metadata, we can't end metadata before upload or there will be script errors. I will add this comment. 

---
In reply to: [368679924](https://github.com/microsoft/clarity-js/pull/166#discussion_r368679924) [](ancestors = 368679924)